nri-elasticsearch.yaml

```diff
@@ -0,0 +1,37 @@
+package:
+  name: nri-elasticsearch
+  version: 5.2.1
+  epoch: 0
+  description: New Relic Infrastructure Elasticsearch Integration
+  copyright:
+    - license: MIT
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - go
+      - build-base
```

Comment:
```suggestion
```

The `go/build` pipeline ensures go is installed (and all the rest), so you don't need to specify these.

As an added bonus, I don't think you actually need `build-base`, which means you won't download `gcc` etc to run this build, which makes it faster and cheaper! 😄